fe/tests/stores/skillStore.test.ts

```diff
@@ -10,57 +10,57 @@ describe('Skill Store', () => {
         // `useStore(pinia)`
         setActivePinia(createPinia())
     })
-
+// 
```

Comment:
Why did you put this whole store in commentary?

Reply:
Most of the skillstore (if not all) has been rewritten, so these tests aren't really useful anymore.
For instance, previously, there was 1 function for creating and updating a new skill. This has been splitted into two functions

Reply:
Alright, thanks for the clarification!